src/entities/DealDiscussions.ts

```diff
@@ -0,0 +1,72 @@
+import { Hash } from "services/EthereumService";
+
+export interface IDiscussion {
+  createdBy: string,
+  createdOn: Date,
+  admins: Array<string>,
+  members: Array<string>,
+  isPublic: boolean,
+  clauseHash: string | null,
+  clauseIdx: number | null,
+  topic: string,
+  replies: number,
+  modifiedAt: Date,
+}
+
+export interface IComment {
+  _id: string,
```

Comment:
why the underscore?

Reply:
I've followed the comment structure of theconvo.space

Reply:
Can you document that for the interface?

Reply:
done